test/functional/feature_backwards_compatibility.py

```diff
@@ -56,6 +57,7 @@ def setup_nodes(self):
         self.add_nodes(self.num_nodes, extra_args=self.extra_args, versions=[
             None,
             None,
+            22000000,
```

Comment:
In a617b9d3b7ffdd7f36e8574602b182edf9d70100 "test: previous releases: add v22.0"

Previously these specifiers matched the client version number, but this does not. 22.0's client version is 220000, but this causes the test framework to search for a directory named 0.22.0 rather than 22.0.

However the client version is used in a few places in the test framework to enable features, so it would be nice if this specifier would match the actual client version. Perhaps this could use a string or tuple that represents the dotted version with a conversion within the test framework to the client version?

Reply:
I think I discovered half-way this commit that 22.0 is not expressed as 22000000, but as 220000. I'll look into that.